examples/suit_update/README.md

```diff
@@ -167,6 +168,15 @@ In this case the RIOT node can be reached from the host using its global address
 `aiocoap-fileserver` is used for hosting firmwares available for updates. Device
 retrieve the new firmware using the CoAP protocol.
 
+*either*:
```

Comment:
This seems unrelated to the goal of this PR, right ?

Reply:
No, the lower commit says to add it to $PATH and this is then used in the test script to setup the fileserver.

Reply:
How is this useful from the user point of view ? except by introducing confusion ? I mean this way of installing is only done for the CI so there's no need to document it. Or maybe only give one way of installing the fileserver ?

Reply:
Hmm you are right, maybe it should be split in test documentation?

Reply:
I changed the prerequisites section to include the wget method, saving to ```~/.local/bin```.
Also added a paragraph with the requirement to add "~/.local/bin" to PATH.